README.md

```diff
@@ -368,6 +369,34 @@ resource_group_name  = "existing_rg"
 virtual_network_name = "existing_vnet"
 ```
 
+**Deploying without Application Gateway**
+
+You can deploy GraphDB without creating Application Gateway in order to use your own. To do that there are some steps to do:
+
+_Prerequisites:_
+Resource Group: A resource group should already be created.
```

Comment:
Can't the Application Gateway be deployed in a different RG ?

Reply:
You can but it becomes different task. Also there is a bit of a problem connecting the application gateway with the vmss.